specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-02-01/management.json

```diff
@@ -0,0 +1,1997 @@
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/getEntities": {
+      "post": {
+        "tags": [
+          "Entities"
+        ],
+        "operationId": "Entities_List",
+        "description": "List all entities (Management Groups, Subscriptions, etc.) for the authenticated user.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SkipTokenParameter"
+          },
+          {
+            "$ref": "#/parameters/SkipParameter"
+          },
+          {
+            "$ref": "#/parameters/TopParameter"
+          },
+          {
+            "$ref": "#/parameters/SelectParameter"
+          },
+          {
+            "$ref": "#/parameters/SearchParameter"
+          },
+          {
+            "$ref": "#/parameters/EntityFilterParameter"
+          },
+          {
+            "$ref": "#/parameters/EntityViewParameter"
+          },
+          {
+            "$ref": "#/parameters/GroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CacheControlHeader"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/EntityListResult"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "GetEntities": {
+            "$ref": "./examples/GetEntities.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/startTenantBackfill": {
+      "post": {
+        "tags": [
+          "TenantBackfill"
+        ],
+        "operationId": "StartTenantBackfill",
+        "description": "Starts backfilling subscriptions for the Tenant.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/TenantBackfillStatusResult"
+            }
+          },
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "#/definitions/TenantBackfillStatusResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "StartTenantBackfill": {
+            "$ref": "./examples/StartTenantBackfillRequest.json"
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
also indicate how it is completed


        "x-ms-long-running-operation-options": {
          "final-state-via": "azure-async-operation"  or "location"
        },

Reply:
Actually removed this 202 response, as it's not valid, we do not return a 202 on this API. added long running operation options to the other async APIs.